pom.xml

```diff
@@ -113,7 +115,13 @@
         <dependency>
             <groupId>pentaho-kettle</groupId>
             <artifactId>kettle-engine</artifactId>
-            <version>7.1-SNAPSHOT</version>
+            <version>9.1.0.0-324</version>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.pentaho.di.plugins</groupId>
+            <artifactId>pdi-core-plugins-impl</artifactId>
+            <version>9.1.0.0-324</version>
```

Comment:
After upgrading mysql connector, I ran into issues with Pentaho 7, which were fixed in Pentaho 9, so I upgraded.  I also found that in Pentaho 9 some of the transforms that were in use in Malawi needed to be pulled in from a separate plugin library, so I tracked that down and added it here.